ras_party/controllers/queries.py

```diff
@@ -0,0 +1,51 @@
+
+from sqlalchemy import func
+from structlog import get_logger
+
+from ras_party.models.models import Business, Respondent
+
+log = get_logger()
+
+
+def query_business_by_party_uuid(party_uuid, session):
```

Comment:
Is 'get' a better word to use than 'query' in this file?

Reply:
I agree, I was going to mention this in another PR but held myself back. I think anywhere we expect to get only one item returned get is more descriptive

Reply:
'get_business_by_ref' and 'get_business_by_id' already exists as business controllers